src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -112,17 +117,8 @@ export class Sash extends EventEmitter {
 		this.$e.addClass('active');
 		this.emit('start', startEvent);
 
-		let overlayDiv = $('div').style({
-			position: 'absolute',
-			top: 0,
-			left: 0,
-			width: '100%',
-			height: '100%',
-			zIndex: 1000000,
-			cursor: this.orientation === Orientation.VERTICAL ? 'ew-resize' : 'ns-resize'
-		});
-
 		let $window = $(window);
```

Comment:
I’ve written some explanations in the corresponding commit message, 0636b55e2ed8422f16ed7196db1ebdf33b6ac5ed. I can explain more if that’s not clear.